jans-cedarling/cedarling/src/authz/entities/test_create.rs

```diff
@@ -42,10 +41,15 @@ fn successful_scenario_empty_namespace() {
         "set_set_key": [["some_string"]]
     });
 
-    let payload: TokenPayload = serde_json::from_value(json).unwrap();
+    let payload: TokenData = serde_json::from_value(json).unwrap();
 
     let entity = metadata
-        .create_entity(&schema, &payload, HashSet::new(), &Default::default())
+        .create_entity(
+            &schema,
+            &Token::Access(payload),
```

Comment:
it is not clear, why this method should know about token type..

Reply:
because we can't put the token type inside the `TokenData` since the `get_record_expression` function creates new `TokenData` inside it. see my above replies.

Reply:
refactored here [18ed767](https://github.com/JanssenProject/jans/pull/10436/commits/18ed7673bc485b13e045d2159b04de12cfc8641a).